server/routes/customerReviews.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```

Comment:
Check linter settings (and consider a precommit hook) - these should be single quotes.

Reply:
My linter somehow broke midway through and i had no idea. Reinstalled it and will look into precommit hooks. Thank you!